timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -786,6 +860,7 @@ template <typename RealType> void declare_nonbonded_all_pairs(py::module &m, con
         m, pyclass_name.c_str(), py::buffer_protocol(), py::dynamic_attr())
         .def("set_nblist_padding", &timemachine::NonbondedAllPairs<RealType>::set_nblist_padding, py::arg("val"))
         .def("disable_hilbert_sort", &timemachine::NonbondedAllPairs<RealType>::disable_hilbert_sort)
+        .def("set_atom_idxs", &timemachine::NonbondedAllPairs<RealType>::set_atom_idxs)
```

Comment:
nit: missing argument name

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/915/commits/ee6008742be225302c0e4fd51cd0a38d884e810b